cmd/triggermesh-controller/main.go

```diff
@@ -167,6 +169,7 @@ func main() {
 		// transformation
 		synchronizer.NewController,
 		transformation.NewController,
+		jqtransformation.NewController,
```

Comment:
Let's preserve the alphabetical order like we do with other components categories.